code/modules/vore/eating/vorepanel_vr.dm

```diff
@@ -378,7 +382,6 @@
 
 	if(href_list["close"])
 		qdel(src)  // Cleanup
```

Comment:
Does qdel on browser objects close them?

Reply:
It does now!